### DISCUSSION TITLE
When resizing or opening a new window tooltip that display size has missing icon in the middle

### DISCUSSION THREAD

**Discussion by kurund** [ghostty-org/ghostty#3485] [Q&A] [answered]
I am on Arch with hyprland

```bash
Ghostty 1.0.1-main+a8e5eef1

Version
  - version: 1.0.1-main+a8e5eef1
  - channel: tip
Build Config
  - Zig version: 0.13.0
  - build mode : builtin.OptimizeMode.ReleaseFast
  - app runtime: apprt.Runtime.gtk
  - font engine: font.main.Backend.fontconfig_freetype
  - renderer   : renderer.OpenGL
  - libxev     : main.Backend.io_uring
  - GTK version:
    build      : 4.16.7
    runtime    : 4.16.7
  - libadwaita : enabled
    build      : 1.6.2
    runtime    : 1.6.2
```

**Comment by mitchellh** [comment]
Can you share a screenshot? I'm not sure what exactly you're referencing here.

**Reply by kurund** [reply]
It's hard to take a screenshot as tool-tip disappears quickly. I tried recording video. Please check the characters as I resize the window.


https://github.com/user-attachments/assets/fd225e02-cf93-4ce3-9fba-6f3b883aa295

**Reply by ridicolos** [reply]
Did you find a way to "disable" this?

**Reply by kurund** [reply]
Nope, I haven't got a chance to look into this yet.

**Reply by 00-kat** [reply]
> "disable" this

@ridicolos What is "this" here?  If you mean the resize overlay, check out [`resize-overlay`].

[`resize-overlay`]: https://ghostty.org/docs/config/reference#resize-overlay

**Reply by ridicolos** [reply]
@00-kat You are correct. I meant this resize-overlay, but could not find it when I last checked. Thank you really much :)

**Comment by jcollie** [comment]
What font is being used? The default Gnome font Cantarell has support for the character, which is U+2A2F vector or cross product.

![image](https://github.com/user-attachments/assets/3f8ea1a6-fa68-49cc-bd74-187ac114a5a6)

**Reply by kurund** [reply]
I am on Hyprland. System wide font is set to `Cantarell`. I changed my GTK Settings to JetBrains Mono too. Same behavior.

**Comment by 00-kat** [comment, marked as answer]
Note: this will be “fixed” by #6040 **if** that is merged.

**Reply by kurund** [reply]
yes, I can confirm that this is resolved now. Thank you
